tests/cell_test.cc

```diff
@@ -1615,10 +1605,10 @@ TEST_CASE("Cell is checked for 3D case", "[cell][3D]") {
     // check if the correct amount of material ids were added to node and if
     // their indexes are correct
     std::vector<unsigned> material_ids = {1, 2};
-    for (int j = 0; j < nodes.size(); ++j) {
-      REQUIRE(nodes[j]->material_ids().size() == 2);
+    for (auto node_itr : nodes) {
```

Comment:
This is not an iterator it's the node so please use for (const auto& node : nodes) {